mautic/mautic
# [UI] Workaround layering model

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR changes the CSS structure related to the layering model and wipes automatic styles that weren't working properly. Also includes a temporary workaround: tab content bg was reset to transparent as fix for the issue mentioned [in Slack](https://mautic.slack.com/archives/C024XBCQS3Z/p1732115616475359) by @laurielim.



| Before                                 | After
| -------------------------------------- | ---
| ![image](https://github.com/user-attachments/assets/d580d637-9fb9-4f63-882a-ba67c2fb3367)                                     | ![image](https://github.com/user-attachments/assets/e58df225-75a9-47b8-85c8-24c716ee487f)


---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Forms
3. Create new
4. See that fields are visible on the screen without background
